README.md

```diff
@@ -144,6 +144,14 @@ If you see this error during an update of the SAM-TRE stack:
 
 Solution: Terminate your EC2 instances named TerraformExecutionInstance. Then rerun the failed command.
 
+#### NOTE: Consider following the below steps if you are attempting to update an already in-use installation
+1. **Manually disable the Lambda subscription to the provisioning intake SQS queues.** This will allow messages to be sent to the queues by Service Catalog but prevent the EC2 instance(s) from starting new workflows that could be affected by shutting down the EC2 instances in following steps
```

Comment:
The terms "trigger" and "event source" are used, but I don't think "subscription is used.

There are two functions that have SQS trigers. List their names here so the user does not need to figure out which functions have SQS triggers: TerraformEngineProvisioningHandlerLambda and TerraformEngineTerminateHandlerLambda

Reply:
Also "in following steps" -- Change to "in the following steps"

Reply:
Agreed on both. Will update in the next revision.